### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to v5.8.7

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -98,10 +101,10 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 ynh_script_progression --message="Setting up application with cURL..."
 
 # Set right permissions for curl install
-chown -R $app: $final_path
+chown -R $app $final_path
```
Comment (yalh76) [inline scripts/install:104]: Not sure i's needed or have to change L77

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — APPROVED** [review]
LGTM

**Pull request merged**